Script-Style Process Types (#21)
paketo-buildpacks/procfile: README.md

The buildpack will do the following:

* Contribute the process types from `Procfile` to the image.

>   * If the application's stack is `io.paketo.stacks.tiny` the contents of the `Procfile` must be single commands with arguments (no inline environment variable declarations, environment variable references, or joined processes) and will be shellparsed to run in `direct` mode.

Reviewer: ```suggestion
  * If the application's stack is `io.paketo.stacks.tiny` the contents of the `Procfile` must be single command with zero or more space delimited arguments. Argument values containing whitespace should be quoted. The resulting process will be executed directly and will not be parsed by the shell.
```

>   * If the application's stack is not `io.paketo.stacks.tiny` the contents of `Procfile` have no limiataions and will be executed in a shell as-is.

Reviewer: ```suggestion
  * If the application's stack is not `io.paketo.stacks.tiny` the contents of `Procfile` will be executed as a shell script.
```
